projects/gvisor/project.yaml

```diff
@@ -0,0 +1,11 @@
+homepage: "https://github.com/google/gvisor"
+main_repo: "https://github.com/google/gvisor"
+primary_contact: "gvisor-dev@googlegroups.com"
+auto_ccs :
+  - "krakauer@google.com"
```

Comment:
I think we talked about my address as `primary_contact` and the mailing list as `auto_ccs` -- this is backwards.

Reply:
Sorry for that. `project.yaml` file has been updated.